### DISCUSSION TITLE
🧩 FEATURE REQUEST: seperate followed tags in a new tab or make followed tabs priority over normal tags

### DISCUSSION THREAD

**Discussion by ariaieboy** [dailydotdev/daily#1093] [Feature Requests]
### Description
right now if we follow a tag we only see posts that have that tag for example if i only follow laravel tag all my feeds will show posts that have laravel tag.
because of that i will always block the tags that i dont want to see instead of following tags that i like

### Solution
the first solution can be to seperate followed tags from non followed tags and make a seperate tab for them sth like instagram we have a feed for our followed users and we have the explore section to find new stuff and follow new users.
for example i use TALL stack (tailwind alpine laravel livewire) for my projects and i like to read related posts to this tools more often and learn more and more about this tools but in my free times i am looking for other good articles related to the development and etc
if we have this feature i can follow my most used tags and see related posts in my feed and in free times i can explore in other areas and learn new things.

### Alternate Solutions
another solution is to prior the followed tags over the non followed tags. i dont know how you rate posts and listing them as popular but you can add a weight system that prior the followed tags over the non-followed so we see more related posts about the followed tags and less from non followed instead of only showing posts with followed tags

**Comment by idoshamun** [comment]
Hey @ariaieboy,

In the upcoming month or two, we will introduce custom feeds. They will allow you to set multiple feeds with different customization so you can follow every topic you want.

**Comment by Julian-Dumitrascu** [comment]
> Hey @ariaieboy,
> 
> In the upcoming month or two, we will introduce custom feeds. They will allow you to set multiple feeds with different customization so you can follow every topic you want.

If you've done it, where are custom feeds described in your user guide?

**Comment by idoshamun** [comment]
@Julian-Dumitrascu unfortunately we haven't done it yet. We've changed our priorities and are working on other new features :)

**Comment by platform-kit** [comment]
@idoshamun please prioritize this. Without filtering, visiting sites I usually get news from independently is a better UX than daily.dev. It should be a core feature of any aggregator.

**Comment by idoshamun** [comment]
Unfortunately, we are a small team and we have other high priority features to work which I hope you will appreciate as well :)
